mobile/e2e-tests/scripts/live_word_monitor.py

```
@@ -1284,230 +1649,237 @@ def evaluate_high_average_latency_incident(self, now_ms: int) -> None:
-                focused_app
-                and "com.mentra.mentra" in focused_app
-                and "MainActivity" in focused_app
-            )
+                    candidate_focus = line.strip()
+                    fallback_focus = candidate_focus
+                    if "null" not in candidate_focus:
+                        focused_app = candidate_focus
+            if focused_app is None:
+                focused_app = fallback_focus
+            is_app_foreground = bool(focused_app and "com.mentra.mentra" in focused_app and "MainActivity" in focused_app)
         except Exception as exc:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)